R/plot.earlyR.R

```diff
@@ -24,7 +24,10 @@
 #'   expected over the time period of the lambdas, or a recognised `character`
 #'   string; lambdas will be scaled to correspond to the number of expected
 #'   cases every day; defaults to `ml`, which tells function to use the maximum
-#'   likelihood estimate of *R* multiplied by the number of infectious cases
+#'   likelihood estimate of *R* multiplied by the number of infectious cases.
+#'
+#' @param colour_spectrum Only applied when plotting "lambdas".  If TRUE a
```

Comment:
The use of colors for the plot of lambdas was a horror I had committed in the initial release. Been cursing myself a lot since. Probably best if we drop it, as it was not adding any functionality, and was a poor use of color :(